### DISCUSSION TITLE
An Invisible notification can not be read?

### DISCUSSION THREAD

**Discussion by yi-Xu-0100** [community/community#45704] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

In my notification, I got an invisible notification, but I can not find it in <https://github.com/notifications> and the reminder badge cannot be canceled, how should I find this notification?

Some of my attempts.

1. I find the invisible notification point to a repository that I watched, and I try to disable the watch, but not helpful.
2. When a new notification comes, the notification will show **<View all 2 notifications>** below the new one. But no new thing when I click it (Only the new one there).
3. when there are two notifications in the page(<https://github.com/notifications>), I think that It should not have the **<View all 2 notifications>** notification but show all (other repository show all). This is not normal.
4. I have used the inprivate mode of the browser to avoid the influence of browser plug-ins, and this situation still exists.

I need help to delete this notification, it keeps reminding me of notifications and I can't deal with it, it makes me anxious. 😥

**Comment by pryley** [comment, marked as answer]
Try this: https://github.com/orgs/community/discussions/6874#discussioncomment-1872442

If that doesn't work, then try this: https://github.com/orgs/community/discussions/6874#discussioncomment-2859125

**Reply by yi-Xu-0100** [reply]
Thanks a lot!

**Reply by femdiya** [reply]
Thank you soo muchhhh
